helix-core/src/main/java/org/apache/helix/model/CloudConfig.java

```diff
@@ -207,6 +207,16 @@ public Builder(CloudConfig cloudConfig) {
       _cloudInfoProcessorName = cloudConfig.getCloudInfoProcessorName();
     }
 
+    public Builder(String cluster, boolean enabled, CloudProvider cloudProvider, String cloudID,
+        List<String> cloudInfoSource, String cloudProcessorName) {
+      _clusterName = cluster;
```

Comment:
nit, let's try to keep the parameter names the same with the fields' names.

Reply:
Fixed.